### DISCUSSION TITLE
How search within one list and show content from another one based on similar column?

### DISCUSSION THREAD

**Discussion by MirasMNC** [microsoft-search/pnp-modern-search#2075] [Q&A]
I have two source lists:
1. List of employees and details about them (name, phone, ID and department). 
2. List of departments and details about them (ID/code, name, activities, description).

Currently I have search based on first list as a phone book that people can search colleagues. Does exist a way that I can connect search results with second list that I can display also details about departments within one search results web part?

Example: I search for contact "John", I got a lot of results and I can see details as number, ID, phone and number of department. I would like to show also details about the department which is different list. I could make a link and redirect people to another page. Ideally I would like to show it within one page, within one results or on hover window.

Does exist a way that I can do search in search based on received results that I use as a keyword the received result? 

I hope it is clear. Maybe somebody has experience with similar case.

**Comment by FranckyC** [comment]
Hi @MirasMNC, maybe this is what your are looking for: https://franckcornu.com/post/modern-search-focus-item-selection/

**Reply by MirasMNC** [reply]
Hi @FranckyC , thank you for your tip. I will try it :)

**Reply by MirasMNC** [reply]
Thank you for you inspiration. I tried it but I got stuck now. I have two lists and I have the connection between them via "lookup" column.
<img width="558" alt="image" src="https://user-images.githubusercontent.com/62887437/167840276-5980f23d-7b24-4867-81ff-2b52c05c4d1b.png">

I added a mapping on this column to RefinableString137.
<img width="1340" alt="image" src="https://user-images.githubusercontent.com/62887437/167840011-e66daa56-8c33-4382-94ea-1d1971bd86fd.png">

But at the end I can not choose this column within search settings. Do you know why? How to get this RefinableString137 to the rollout menu?
<img width="565" alt="image" src="https://user-images.githubusercontent.com/62887437/167840224-53c6fd6d-c5cb-4557-85cb-bb7bbd9786b9.png">

**Reply by MirasMNC** [reply]
Hi @FranckyC , do you have any idea how to get custom refinable to the list for filtering? Thanky you :)

**Reply by gretchunkim** [reply]
@MirasMNC : Is this resolved issue for you? I believe you'll need to wait crawl/force crawl to make it appear on the Field section on the webpart.
